From: Emmanuele Bassi <ebassi@gnome.org>
Date: Tue, 23 Jul 2019 11:17:41 +0100
Subject: docs: Remove mentions of mailing lists

And drop FTP from the list of places where to find release archives.
This is not 1998.

Origin: upstream, 2.61.3, commit:ab544aa8c0037e8abb7c1c5617fffd8efa5e5ae1
---
 README.md | 20 ++++++--------------
 1 file changed, 6 insertions(+), 14 deletions(-)

diff --git a/README.md b/README.md
index 677f2dd..302d5ac 100644
--- a/README.md
+++ b/README.md
@@ -1,24 +1,16 @@
 General Information
 ===================
 
-GLib is the low-level core
-library that forms the basis for projects such as GTK+ and GNOME. It
-provides data structure handling for C, portability wrappers, and
-interfaces for such runtime functionality as an event loop, threads,
-dynamic loading, and an object system.
+GLib is the low-level core library that forms the basis for projects such
+as GTK and GNOME. It provides data structure handling for C, portability
+wrappers, and interfaces for such runtime functionality as an event loop,
+threads, dynamic loading, and an object system.
 
 The official download locations are:
-  ftp://ftp.gtk.org/pub/glib
-  http://download.gnome.org/sources/glib
+  https://download.gnome.org/sources/glib
 
 The official web site is:
-  http://www.gtk.org/
-
-Information about mailing lists can be found at
-  http://www.gtk.org/mailing-lists.php
-
-To subscribe, send mail to gtk-list-request@gnome.org
-with the subject "subscribe".
+  https://www.gtk.org/
 
 Installation
 ============
